bootstrap.d/media-libs.y4.yml

```diff
@@ -40,6 +40,101 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: libpulse
+    architecture: '@OPTION:arch@'
+    metadata:
+      summary: Libraries for PulseAudio clients
+      description: Libraries for PulseAudio clients.
+      spdx: 'LGPL-2.1+'
+      website: 'https://www.freedesktop.org/wiki/Software/PulseAudio'
+      maintainer: "Dennis Bonke <dennis@managarm.org>"
```

Comment:
Should this be Dennis? I think you can also just add yourself, Qwinci.

Reply:
Yeah qwinci can add himself here but if he wants to pass maintainership to me that's fine too.